### PR TITLE
Fix command used for enable service catalog in docs

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: docs/source/using/experimental-features.adoc [hunk 1]
```diff
@@ -33,10 +33,8 @@ You can set the `MINISHIFT_ENABLE_EXPERIMENTAL` environment variable to enable t
 `extra-clusterup-flags`::
 Enables passing flags directly to `oc cluster up` that are not directly exposed in the {project} CLI.
 
-For example, the following command will pass the `--service-catalog` flag to `oc cluster up` to provision the OpenShift link:https://docs.okd.io/latest/architecture/service_catalog/index.html[*service catalog*]:
-
 ----
-$ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--service-catalog"
+$ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog'"
```
Comment (anjannath) [inline docs/source/using/experimental-features.adoc]: This method works for okd v3.10 the earlier one works for v3.9 we need to mention that and keep both the ways in the docs.

Comment (geoand) [reply docs/source/using/experimental-features.adoc]: Good point! I will update it

**Comment by LalatenduMohanty** [comment]
@geoand We need issues for each PR as per our contributor guidelines [1]. Hence the CI is not passing for Circleci. However if you want I can fix this for you and merge the patch. Let me know. 

[1] https://github.com/minishift/minishift/blob/master/CONTRIBUTING.adoc#contributing-patches

File: docs/source/using/experimental-features.adoc [hunk 2]
```diff
@@ -33,7 +33,13 @@ You can set the `MINISHIFT_ENABLE_EXPERIMENTAL` environment variable to enable t
 `extra-clusterup-flags`::
 Enables passing flags directly to `oc cluster up` that are not directly exposed in the {project} CLI.
 
-For example, the following command will pass the `--service-catalog` flag to `oc cluster up` to provision the OpenShift link:https://docs.okd.io/latest/architecture/service_catalog/index.html[*service catalog*]:
+Using the experimental flag, to enable the Service Catalog in OKD 3.10 the command is:
```
Comment (LalatenduMohanty) [inline docs/source/using/experimental-features.adoc]: Please replace `OKD 3.10` with `OKD 3.10.0 or later`

**Comment by geoand** [comment]
@LalatenduMohanty PR rebased and contains the fixes you mention

**Comment by LalatenduMohanty** [comment]
@geoand Thank you.

**Comment by geoand** [comment]
@LalatenduMohanty you are very welcome!

**Pull request merged**